docs/source/_templates/classtemplate.rst

```diff
@@ -0,0 +1,14 @@
+.. role:: hidden
+    :class: hidden-section
+.. currentmodule:: {{ module }}
+
+
+{{ name | underline }}
+
+.. autoclass:: {{ name }}
+    :members:
+
+
+..
+  autogenerated from source/_templates/classtemplate.rst
+  note it does not have :inherited-members:
```

Comment:
This is the template for `autoclass` to appear like pytorch docs.
without this, the `h1` will be `module.{module}.class`, which is long for sub packages.